src/server/debug/server/util.go

```diff
@@ -132,3 +134,59 @@ func copyTar(tw *tar.Writer, tr *tar.Reader, prefix ...string) error {
 		}
 	}
 }
+
+func validateProject(p *pfs.Project) error {
```

Comment:
Don't we have a `ppsutil` or `pfsutil` where these could be exported.  It seems like they would get a lot of use?

Reply:
There is a `src/internal/ppsutil`, but no `pfsutil` (yet). If we did end up needing them anywhere else in the codebase then it would probably make sense to put them in a new `src/internal/pfsutil`, but since we don’t yet I think it’s better not to add more entities to the system.